packages/thirdweb/src/wallets/smart/lib/userop.ts

```diff
@@ -147,6 +147,9 @@ export async function createUnsignedUserOp(args: {
     partialOp.preVerificationGas = estimates.preVerificationGas;
   }
 
+  // add 50k overhead for entrypoint checks (following same as unity-sdk)
+  partialOp.callGasLimit = partialOp.callGasLimit + 50_000n;
```

Comment:
same here - there's diff path including one that needs to re-sign the payload from the paymaster